src/types/Strings.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Special String Types
+ *
+ * @packageDocumentation
+ */
+
+/**
+ * 0x prefixed Hexadecimal String
+ */
+export declare type HexString = `0x${Lowercase<string>}`;
+
+/**
+ * 0x prefixed Hexadecimal representation of an Ethereum Address
+ */
+export declare type EthereumAddress = HexString;
+
+/**
+ * 0x prefixed Hexadecimal representation of an Ethereum Contract that implements the Social Identity Interface
+ */
+export declare type SocialIdentityAddress = EthereumAddress;
```

Comment:
There are three types of ethereum addresses. All lower, all upper (equivalent) and mixed for https://eips.ethereum.org/EIPS/eip-55

Currently HexString only claims one, but we don't really have much control over which we are using.